package.json

```diff
@@ -30,7 +30,7 @@
     "preact-cli": "^3.0.0-rc.7",
     "preact-render-spy": "^1.2.1",
     "react-syntax-highlighter": "^12.2.1",
-    "webpack-bundle-analyzer": "^3.4.1"
+    "webpack-bundle-analyzer": "^3.4.1",
```

Comment:
Minor nit: That comma makes the json file invalid as the json spec doesn't allow trailing commas :+1:

Reply:
You right! I will fix that